pallets/utxo/rpc/runtime-api/src/lib.rs

```diff
@@ -25,6 +26,8 @@ sp_api::decl_runtime_apis! {
         // At the moment we have some problems with use serde in RPC, we can serialize and deserialize
         // only simple types. This approach allow us to return Vec<(TokenId, TokenName)> instead of
         // pallet_utxo_tokens::TokenListData
-        fn tokens_list() -> Vec<(u64, Vec<u8>)>;
+        fn tokens_list() -> Vec<(H256, Vec<u8>)>;
+        // Getting NFT data
+        fn nft_read(id: H256) -> Option<(/* Data url */ Vec<u8>, /* Data hash */ [u8; 32])>;
```

Comment:
I think limiting the data to 32 bytes is a big mistake. We should have a serialization mechanism that determines the data size. The simplest, is the first N bytes (4 is good enough), encoded with little or big endian, determining the size of the data. Then it can be as long as the user wants. The bigger it's, the more expensive the transaction will be.